wolfcrypt/src/asn.c

```diff
@@ -20696,6 +21275,100 @@ int wc_GetPubKeyDerFromCert(struct DecodedCert* cert,
     return ret;
 }
 
+#ifdef WOLFSSL_FPKI
+/* Search through list for first matching alt name of the same type
+ * If 'current' is null then the search starts at the head of the list
+ * otherwise the search starts from the node after 'current' alt name.
+ * Returns 0 on success
+ */
+static DNS_entry* wc_GetAltName(struct DecodedCert* cert, int nameType,
```

Comment:
Seems odd to use a public named API `wc_GetAltName` for a static. Can you rename to something else?